pkg/app/app.go

```diff
@@ -1062,7 +1062,13 @@ func (a *App) visitStatesWithSelectorsAndRemoteSupport(fileOrDir string, converg
 		}
 	}
 
-	return a.visitStates(fileOrDir, opts, f)
+	// pre-overrides HelmState
+	fHelmStatsWithOverrides := func(st *state.HelmState) (bool, []error) {
+		st.Releases = st.GetReleasesWithOverrides()
```

Comment:
How does this resolve the issue? 🤔
The error message looked as if it was saying that the chart name that looks like a URL isn't valid. And this GetReleasesWithOverrides doesn't touch the chart name of the problematic release, right?

Reply:
pre handle `st.Releases` by `GetReleasesWithOverrides` func. So top namespace will be set correctly. GetReleasesWithOverrides can be called many times, it is idempotence for new code.